### ISSUE TITLE
Closing/minimizing a window will trigger the Dock bar to accidentally bump out (when the Dock bar is set to auto-hide)

### ISSUE THREAD

**Issue by JDDPro** [mydockfinder/mydockfinder#689]
Closing/minimizing a window will sometimes trigger the Dock bar to accidentally bump out (when the Dock bar is set to auto-hide). This doesn't happen everytime a window is closed/minimized, but has a high chance of happening - I'd say the chance is 50/50.

关闭或者最小化一个窗口，有时候会错误触发停靠栏（Dock bar）自动跳出来（当停靠栏设置为自动隐藏的时候）。这种情况并不是每次关闭或者最小化窗口都会发生，但是发生的几率还是挺高的——我觉得差不多有一半的概率会这样。

**Comment by JDDPro** [comment]
Also: The dock bar will jump out when opening a new window (sometimes) and switching between virtual desktops (almost every time)

**Comment by mydockfinder** [comment]
Yes, update fix today




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月13日(星期六) 下午5:27
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Closing/minimizing a window will trigger the Dock bar to accidentally bump out (when the Dock bar is set to auto-hide) (Issue #689)





 
Also: The dock bar will jump out when opening a new window (sometimes) and switching between virtual desktops (almost every time)
 
&mdash;

**Comment by JDDPro** [comment]
> Yes, update fix today
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2024年4月13日(星期六) 下午5:27 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; 主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Closing/minimizing a window will trigger the Dock bar to accidentally bump out (when the Dock bar is set to auto-hide) (Issue #689) Also: The dock bar will jump out when opening a new window (sometimes) and switching between virtual desktops (almost every time) &mdash; Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

Thank you! 

When will the fixed version be released? 

The issue is still present in the latest version downloaded from Steam (1.9.9.8).

**Comment by JDDPro** [comment]
I figure that the Dock problem is now mostly triggered by closing/opening windows. Minimizing windows now hardly makes it happen.

**Comment by mydockfinder** [comment]
Yes, especially when closing the window, I found a pattern, is it that the system doesn't shift the focus of the window to the new window after closing the window, or is it the hidden window that makes the dock show up automatically, I need to fix this slowly




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月16日(星期二) 上午6:12
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Closing/minimizing a window will trigger the Dock bar to accidentally bump out (when the Dock bar is set to auto-hide) (Issue #689)





 
I figure that the Dock problem is now mostly triggered by closing/opening windows. Minimizing windows now hardly makes it happen.
 
&mdash;

**Comment by JDDPro** [comment]
Not very sure but this seems to be resolved in version 1.9.9.8

I'll close this issue soon if the fix is confirmed.

**Comment by mydockfinder** [comment]
This issue is not 100% fixed and will be fixed in the next release
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月16日(星期四) 下午5:10
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Closing/minimizing a window will trigger the Dock bar to accidentally bump out (when the Dock bar is set to auto-hide) (Issue #689)














 
Not very sure but this seems to be resolved in version 1.9.9.8
 
I'll close this issue soon if the fix is confirmed.
